### ISSUE TITLE
window.FB not defined

### ISSUE THREAD

**Issue by nickgermaine** [keppelen/react-facebook-login#220]
Admittedly, I'm probably just thick here, but I'd think react-facebook-login would be able to load the SDK or something (like a one-stop solution), but from what I can tell: "nope".

So I've got some code here, basically, I'm trying to load in the js sdk on my own in my constructor.  Still, though, window.FB is undefined.  In fact, those console statements in the init aren't even firing so I'm positive I'm just thick right now.  Still, if anyone can see an obvious error, it'd be great to get some help.

constructor(props){
...

window.fbAsyncInit = function() {
            FB.init({
                appId            : 'myid',
                autoLogAppEvents : true,
                xfbml            : true,
                version          : 'v2.12'
            });

            // Not even firing off here.....
            console.log("Loading FB", FB, window.FB);
            window.FB = FB;
        };

        (function(d, s, id){
            var js, fjs = d.getElementsByTagName(s)[0];
            if (d.getElementById(id)) {return;}
            js = d.createElement(s); js.id = id;
            js.src = "https://connect.facebook.net/en_US/sdk.js";
            fjs.parentNode.insertBefore(js, fjs);
        }(document, 'script', 'facebook-jssdk'));
}

**Comment by nickgermaine** [comment]
Made progress by npm install fb, that lets me import FB and I set window.FB = FB, so now the error is 
TypeError: window.FB.login is not a function

**Comment by nickgermaine** [comment]
So my code above was actually perfect, which is why I was so confused.  I'm a full stack developer with 10 years of experience who was having a bad day.  Figured I'd update and close this issue.

Turns out, Firefox synced my extensions so there was this extension called Disconnect enabled, which prevents tracking scripts from running, and that was disallowing the facebook sdk from initializing properly.  Once I disabled that, react-facebook-login began working.

I spent a day and a half trying everything I could think of to troubleshoot this as a programming problem, not an environment problem.
